playground/html/index.html

```diff
@@ -8,3 +8,11 @@ <h1>Hello</h1>
 <link rel="icon" href="{{cdn_host}}/cdn/images/favicon.ico" />
 <link rel="stylesheet" href="{{cdn_host}}/css.css" type="text/css" />
 <script src="{{cdn_host}}/js.js"></script>
+
+<!-- accepted HTML parsing errors, browsers accept these -->
+<div foo="a" foo="b">Accept duplicated attribute</div>
+
+<!-- accepted HTML constructs -->
+<datalist>
+  <option value="option-without-closing-tag"></option>
```

Comment:
I guess this has been fixed by Prettier?

Reply:
Oh, no... 😅

Reply:
Resolved in [vitejs/vite@`81b4123` (#9678)](https://github.com/vitejs/vite/pull/9678/commits/81b4123778c0a105667a864c391c1374268fd12d), and added a test